src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -258,7 +258,7 @@ func (a *StorageMinerActorCode_I) _claimDealPayments(rt Runtime) {
 //       - Failing / Recovering / Active / Committed -> Cleared
 //     - Remove SectorNumber from Sectors, ProvingSet
 // - Update ChallengeEndEpoch
-func (a *StorageMinerActorCode_I) _onSuccessfulPoSt(rt Runtime, onChainInfo sector.OnChainPoStVerifyInfo) InvocOutput {
+func (a *StorageMinerActorCode_I) _onSuccessfulPoSt(rt Runtime) InvocOutput {
 	h, st := a.State(rt)
 
 	// TODO add info on chain
```

Comment:
If this TODO will require at least some parameter, I suggest we add that now rather than over-simplify.

For context, I need to thread whatever is required here from the block headers through the interpreter to construct the implicit EPost processing method.

Reply:
Agree. Halted this PR after I read through more carefully (but not before i put it up. Derp)